### PR TITLE
Ft 208 replay

### PR DESCRIPTION
**Pull request by sfranky**

### CODE REVIEW

File: qtop.py [hunk 1]
```diff
@@ -1937,24 +1941,6 @@ def colorize(text, color_func=None, pattern='NoPattern', mapping=None, bg_color=
         return text
 
 
-def parse_user_replay_duration(duration):
```
Comment (sfranky) [inline qtop.py]: this was renamed to parse_time_input and moved. It's now used twice, too !!

File: utils.py [hunk 2]
```diff
@@ -78,11 +78,17 @@ def parse_qtop_cmdline_args():
...
-    parser.add_option("-R", "--replay", action="append", type="string", dest="REPLAY", default=[],
-                      help="instant replay from a specific moment in time for the cluster, and for a specific duration. "
-                           "The value inputted should be of the form <yyyymmddThhmmss>, e.g. -R 20161117T102300."
-                           "A second value is for now mandatory and denotes the desired length of the playback in minutes,"
-                           "e.g. -R 20161117T102300 -R2")
+    parser.add_option("-R", "--replay", action="callback", dest="REPLAY", callback=_watch_callback,
+                      help="instant replay from a specific moment in time for the "
+                           "cluster, and for a specified duration. The value "
+                           "inputted should be in either of the following formats: "
+                           "yyyymmddTHHMMSS, e.g. 20161118T182300, (explicit form) "
```
Comment (fgeorgatos) [inline utils.py]: so... this is indeed a very good PR.

I propose a filtering of some chars to make this expression more generic, fi: `''.join([x for x in a if x not in ':/-'])`

Comment (sfranky) [inline utils.py]: added

**Comment by fgeorgatos** [comment]
meerrrrggggiiing!!!

**Pull request merged**